server/src/com/thoughtworks/go/server/controller/GoConfigAdministrationController.java

```diff
@@ -105,18 +73,6 @@ public void getConfigRevision(@RequestParam(value = "version", required = true)
         XmlAction.xmlFound(configRevision.getContent(), md5).respond(response);
     }
 
-    @RequestMapping("/admin/restful/configuration/group/GET/xml")
-    @Deprecated
-    public void getGroupAsXmlPartial(@RequestParam("pipelineGroup")String groupName,
-                                     @RequestParam(value = "md5", required = false)String md5,
-                                     HttpServletResponse response) throws Exception {
-        if (isTemplate(groupName)) {
-            getXmlPartial(groupName, md5, goConfigService.templatesSaver()).respond(response);
-            return;
-        }
-        getXmlPartial(groupName, md5, goConfigService.groupSaver(groupName)).respond(response);
-    }
-
```

Comment:
Can you check if the methods on GoConfigService are used? `buildSaver` `stageSaver` and the likes?